tests/acceptance/features/apiSpacesShares/shareUploadTUS.feature

```diff
@@ -205,13 +205,13 @@ Feature: upload resources on share using TUS protocol
     Given user "Alice" has created folder "/FOLDER"
     And user "Alice" has shared folder "/FOLDER" with user "Brian"
     And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
-    When user "Brian" creates a new TUS resource for the space "Shares" using the WebDAV API with these headers:
+    When user "Brian" creates a new TUS resource for the space "Shares" with content "" using the WebDAV API with these headers:
       | Upload-Length   | 16                                    |
       #    L0ZPTERFUi90ZXh0RmlsZS50eHQ= is the base64 encode of /FOLDER/textFile.txt
       | Upload-Metadata | filename L0ZPTERFUi90ZXh0RmlsZS50eHQ= |
       | Tus-Resumable   | 1.0.0                                 |
     And user "Brian" uploads file with checksum "MD5 827ccb0eea8a706c4c34a16891f84e8c" to the last created TUS Location with offset "0" and content "uploaded content" via TUS inside of the space "Shares" using the WebDAV API
-    Then the HTTP status code should be "406"
+    Then the HTTP status code should be "460"
```

Comment:
is that true? what kind of funny code is that, I've never seen 460 before and it is also not mentioned here https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses

Reply:
I thing it is typo because in the opened issue and docs is `460` error. https://github.com/owncloud/ocis/issues/1755

Reply:
@butonic mentions 460 in his PR https://github.com/cs3org/reva/pull/1400/files#diff-198f1004a921b3627f7572a239452974429a7de6e4fa47f445c2ad35d2cd9026R335
So I guess 460 is the correct response code